docs/source/workflow_inputs.rst

```diff
@@ -2143,6 +2144,27 @@ If not entered, the simulation will not include generators.
 
   Generators will be modeled as operating continuously (24/7).
 
+HPXML Batteries
+***************
+
+Each battery is entered as a ``/HPXML/Building/BuildingDetails/Systems/Batteries/Battery``.
+If not entered, the simulation will not include batteries.
+
+  ==========================  =======  =======  ===========  ========  =======  ============================================
+  Element                     Type     Units    Constraints  Required  Default  Notes
+  ==========================  =======  =======  ===========  ========  =======  ============================================
+  ``SystemIdentifier``        id                             Yes                Unique identifier
+  ``BatteryType``             string            See [#]_     Yes                Battery type
+  ``Location``                string            See [#]_     No        outside  Location
+  ``LifetimeModel``           string            See [#]_     No        None     Lifetime model
```

Comment:
Many users will be unfamiliar with this term. I suggest adding a `[#]_` after "Lifetime model" and adding a footnote w/ some language from [the EnergyPlus documentation](https://bigladdersoftware.com/epx/docs/9-6/input-output-reference/group-electric-load-center-generator.html#field-electrical-buss-type).